interfaces/automated-testing/scd/scd_injection_observation.yaml

```diff
@@ -0,0 +1,792 @@
+        contents of this data structure are generated by the client.
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityOVN'
+      default: [ ]
+    DeleteTestResponse:
+      description: ''
+      required:
+        - operational_intents
+      type: object
+      properties:
+        operational_intents:
+          description: 'The complete set of test data deleted.'
+          type: array
+          items:
+            $ref: '#/components/schemas/OperationalIntent'
+
+
+    CreateTestParameters:
+      description: Collection of fully-defined operational intents
+      type: object
+      properties:
+        test_id: 
+          description: ID provided when a test payload is submitted to the system under test
+          type: string
+          format: uuid 
+        operational_intents:
+          description: A list of operational intents provided for the test
+          $ref: '#/components/schemas/OperationalIntentList'
+
+    FlightNotAuthorizedResult:  
+      type: object
+      properties:
+        scd_processing_result:  
+          type: string        
+          enum: [OperationalConflict, 'CouldNotProcess']   
+          description: >
+              Sort order:
+                * `OperationalConflict` - The Flight not Authorized becuase of a conflict with another operation
+                * `CouldNotProcesss` - The OperationalIntent could not be processed by the USSP, this might be because of a number of reasons e.g. the intent is so far in the future the USSP cannot process it.
+        area_of_interest:
+          type: object
+          $ref: '#/components/schemas/Volume4D'
+                  
+      
+    SCDProcessingResults: 
+      description: Results for the operational intents submitted to the create test endpoint
+      type: array
+      items:
+        anyOf:
+          - $ref: '#/components/schemas/OperationalIntent'
+          - $ref: '#/components/schemas/FlightNotAuthorizedResult'
+
+    QuerySCDParameters:
+      description: |-
+        Parameters for a request to find results of strategic de-confliction processing matching the provided criteria.
+      type: object
+      properties:
+        area_of_interest:
+          $ref: '#/components/schemas/Volume4D'
+
+
+
+paths:
+  /scd_tests/{test_id}:
+    parameters:
+      -
+        examples:
+          default:
+            value: '2979bd18-7f06-441c-bda6-e82c841c35d6'
+        name: test_id
+        description: >-
+          The ID of the test.
+        schema:
+          type: string
+        in: query
+        required: true
+    put:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateTestParameters'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreateTestParameters'
```

Comment:
The request and response data types [should not be the same](https://github.com/interuss/dss/pull/554#discussion_r718958855), and this certainly isn't the response data type we want because the USS may not be able to create the specified operational intents.

Reply:
OK I have now changed the Test create data set and additionally a new object to display response